### ISSUE TITLE
Body & Face Spikes coused by Demonic Corruption/Fel

### ISSUE THREAD

**Issue by Remerod** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2#646]
**Describe your suggestion in full detail below:**
So, Fel are known to cause several different mutation on those who practice it or are demonified by it. One Such mutation are Demonic Spikes. with the addition to the mod such as fel orcs, Nathrezim and Man'ari Eredar, I think if we could add some spikes customization if will greatly flavourise these demons. 
I am Suggesting that we should add customization to add demonic Spikes customization options to have as beards, eyebrows & possible for the head/hair section. Also some for the body, on places like; the hands, arms, and the Back. 

I'll post some pictures for the themes of how they could look like, and for whom they might fit.

**Eredar**
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/79544635/131417010-1b834b21-4726-4728-946c-07926b1bb513.png)
![image](https://user-images.githubusercontent.com/79544635/131418979-15bc40f8-17b7-4fb2-ba28-50ec939ca6bf.png)
![image](https://user-images.githubusercontent.com/79544635/131419109-dd37d7bb-70bb-4371-8941-d375d99ff6b8.png)
Kil'Jaeden got spikes all over the place.

![image](https://user-images.githubusercontent.com/79544635/131419457-319c326b-349f-4a27-9294-1c85c6864f39.png)
Fem-Kil'Jaeden Fan art, not canon but the spikes looks cool, so it show it could look nice on female eredar to

![image](https://user-images.githubusercontent.com/79544635/131419134-a5de7540-cc46-4a1c-84d2-9cc89ec42395.png)
Full crazy with Spikes. 

![image](https://user-images.githubusercontent.com/79544635/131419567-d2be365c-ba47-460e-a647-7e44d4591a3d.png)
Beard & Eyebrows Spikes.

![image](https://user-images.githubusercontent.com/79544635/131417038-1ce56a6e-2e91-4217-824f-102882d07463.png)
Beard spikes

![image](https://user-images.githubusercontent.com/79544635/131419298-05e41d1b-5e4f-40a3-8042-950fb3c70b19.png)
Eyebrows Spikes & tiny beard spikes.

</details>

**Nathrezim** 
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/79544635/131417160-5ee6229e-2619-43b4-a8e2-c99254e5ff71.png)
Some chin-beard-spikes & Eyebrows Spikes

![image](https://user-images.githubusercontent.com/79544635/131417234-8a80a269-44d6-4230-bfad-80ee38d02096.png)
Some failed grown spike-area as eyebrows.

![image](https://user-images.githubusercontent.com/79544635/131417309-2c48b09c-1d23-4e75-94c3-b1d3bbf1e49e.png)
![image](https://user-images.githubusercontent.com/79544635/131417330-ee7a2f07-bf9e-4c6b-8523-79c167e9098f.png)
Beard spikes and hair spikes?

![image](https://user-images.githubusercontent.com/79544635/131417364-fe4b519f-33ed-4efd-9901-a2c5ca34414c.png)
Hair Spikes

</details>

**Demonic influenced Orc & Fel Orcs**
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/79544635/131417410-861ef00d-b035-4e96-9288-184474d6eca8.png)
![image](https://user-images.githubusercontent.com/79544635/131417425-0de6e784-05c7-44f7-b69f-b460569e4c5e.png)
Spikes on Hand, Arms and Back.

![image](https://user-images.githubusercontent.com/79544635/131417545-aa6cc50a-3513-4287-9e77-22677874f2eb.png)
![image](https://user-images.githubusercontent.com/79544635/131417576-79ea0966-379f-406d-bd0e-3554f58b9418.png)
![image](https://user-images.githubusercontent.com/79544635/131417587-5e935b07-4b72-40bd-ba83-7bb792bb0d6c.png)
Eyebrows & Hand Spikes, with failed spike growth on arm.

![image](https://user-images.githubusercontent.com/79544635/131417721-dde04034-3eff-4670-bbef-24ff4ed99f31.png)
![image](https://user-images.githubusercontent.com/79544635/131417788-20fd316c-4f99-422f-801c-0581c4dc815b.png)
![image](https://user-images.githubusercontent.com/79544635/131417811-78fd9feb-31b5-4634-94a9-bd78eabf0ba1.png)
Beard & Eyebrows Spikes. With failed spike growth on the side of the Head. Hand & Arm Spikes and Back Spikes.

![image](https://user-images.githubusercontent.com/79544635/131418001-dd038000-f996-4bd8-bc69-90f3871c656e.png)
![image](https://user-images.githubusercontent.com/79544635/131418028-2b3f8516-d094-4475-a2a5-c484c882264c.png)
Arm, Shoulder & Eyebrow Spikes. Failed spike growth on the Arms, Sides on the Chest & on the back/shoulder behind the head.

![image](https://user-images.githubusercontent.com/79544635/131418275-6c828547-3cda-49c7-aa68-b3d4c0ef1a96.png)

![image](https://user-images.githubusercontent.com/79544635/131418529-4c492c51-c0c0-47e5-913e-30a2dd61fc9c.png)
Spikes on hand-knuckles and arm.

![image](https://user-images.githubusercontent.com/79544635/131418654-3d6b7f91-7355-4366-967b-5d84dfc7c058.png)
Hand & Arm spikes

![image](https://user-images.githubusercontent.com/79544635/131418757-378e4b78-8ed1-4664-b51c-ade36df2f5c1.png)
![image](https://user-images.githubusercontent.com/79544635/131418781-68677828-f5a5-4bca-9232-dba2ae3578f0.png)
![image](https://user-images.githubusercontent.com/79544635/131418804-0ca30bb2-43ff-47b5-a171-c36260ba6ac4.png)
Eyebrows & Arm Spikes, and the great Back Spikes of Gul'dan.

![image](https://user-images.githubusercontent.com/79544635/131420008-3bf77a69-4635-4402-82df-4993dd3e4f3b.png)
Beard Spikes

_So yes this is alternative draenor, But i think their spikes formation is a good formation (although they mostly are to big in my opinion)_

![image](https://user-images.githubusercontent.com/79544635/131420121-38266208-6b59-4e00-8edf-33c1228e2624.png)
Shoulder's, Beard & Hair spikes.

![image](https://user-images.githubusercontent.com/79544635/131420247-e79ca2ac-0b0b-4dd1-8d00-f16e38c36910.png)
Arm spikes.

![image](https://user-images.githubusercontent.com/79544635/131420301-10c5525b-ae46-4677-8437-3f7f97cef2bc.png)
Beard and Hair spikes.

![image](https://user-images.githubusercontent.com/79544635/131420480-d0f8f5c2-f03e-4373-87fd-86e787cf26ba.png)
Small/tiny shoulder & beard spikes.

_Some fan art concept of fel orcs (with spikes)_
![image](https://user-images.githubusercontent.com/79544635/131420618-16f78f59-7411-48c8-aff3-0272b6e534b0.png)
Female beard spikes.

![image](https://user-images.githubusercontent.com/79544635/131420653-c1af6c23-1d87-4442-b8c2-cb35320dd7b7.png)
female spikes.

![image](https://user-images.githubusercontent.com/79544635/131420735-92f1a932-e46d-49df-bacc-8afd8f23812d.png)
fel orc with spikes plus hair.

![image](https://user-images.githubusercontent.com/79544635/131420770-eec85a68-dbcd-4d70-ac15-6fdd15567ccf.png)
Spikes all over.

</details>

I also what to point out that i don't think that every one of the demons and orc, that i have mentioned should have spikes, but a medium or small & perhaps should, like it could be common for fel orcs, a medium amount of eredar might have spikes, and only a small percentages of Nathrezim should have spikes, becouse it's rare among them.
_And yes, i know a lot of pictures from wc3 reforged, but they got a loot of nice spikes!_

If you agree, disagree or have an opinion/idea of your own, please leave a comment. ;)

**Comment by ercarp** [comment]
I agree with adding spikes, and I believe @robmart was planning on having such added for the final stage of Fel corruption. However, some of these depictions (especially the ones in Reforged where they have clusters of small spikes next to each other) can be a bit uncomfortable to look at for those with trypophobia (myself included). 😅

Here are my favorites from the artwork you showcased:

<details><summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/78575425/132040204-e4b68066-0aca-4e35-b61f-50560fd34faf.png)
![image](https://user-images.githubusercontent.com/78575425/132040237-eb4b96fb-975a-4cb6-8620-875efa3ccf22.png)
![image](https://user-images.githubusercontent.com/78575425/132040250-852a692c-8e9e-4759-b74e-e5d34f764752.png)
![image](https://user-images.githubusercontent.com/78575425/132040262-8494cf05-3f9a-4870-992c-45ce0c023e69.png)
![image](https://user-images.githubusercontent.com/78575425/132040276-cd4fc0d5-2b3e-4080-9ae7-2c08f34d5b30.png)
![image](https://user-images.githubusercontent.com/78575425/132040291-6f8c8fdd-cb03-400d-bb8c-818552ca2c63.png)
![image](https://user-images.githubusercontent.com/78575425/132040325-112d1b7f-251d-498a-89a8-727de4eac763.png)
![image](https://user-images.githubusercontent.com/78575425/132040333-cbcbc900-db71-4c94-bd23-02a817bd17f6.png)
![image](https://user-images.githubusercontent.com/78575425/132040397-f8f98ff3-9dba-4294-8b34-77b6a0862a7c.png)
![image](https://user-images.githubusercontent.com/78575425/132040463-adb9ac80-b437-41da-bf1e-ab51ef370c33.png)
![image](https://user-images.githubusercontent.com/78575425/132040482-7bf99a14-4ab3-4422-aa52-64cfa04b1054.png)
![image](https://user-images.githubusercontent.com/78575425/132040516-fcb2291e-5182-40ab-84ce-243a39765fe7.png)

</details>

Basically *most* of them are fine and I agree with, I would just avoid the large clusters of small spikes, I think they're a bit much.

**Comment by Remerod** [comment]
I am very unsure what you mean by getting trypophobia from the spikes @ercarp . I really dislike trypophobia to, but i don't see the phobia of wholes here, since they are spikes coming out? could you show me an example to clarify what you mean?
